### DISCUSSION TITLE
pypi installation of highspy on windows machines

### DISCUSSION THREAD

**Discussion by FabianHofmann** [ERGO-Code/HiGHS#1101] [Q&A]
Is there a perspective to enable the highspy installation on windows machines anytime soon? Thanks in advance!

**Comment by jajhall** [comment]
highspy does install on some Windows builds - I've done it on my own laptop

As with many things, Windows is rather flaky, so I can believe that it doesn't install on all Windows builds yet. Quite how we address this is a good question.

**Comment by jajhall** [comment]
@galabovaa has made progress on the Windows installation this week. Maybe give it another try
